classes/book.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'item'
+
+class Book < Item
+  attr_accessor :id, :publisher, :cover_state, :title
+
+  def initialize(title:, publisher:, cover_state:, publish_date:, id: SecureRandom.random_number(1..1000))
+    super(publish_date)
+    @id = id
+    @publisher = publisher
+    @cover_state = cover_state
+    @title = title
+  end
+
+  # If I need to print the book.
```

Comment:
Please remember we need to keep our code clean and the rule YAGNI (You Aren't Gonna Need It). For this specific task we should stick to the UML diagram and not add unnecessary methods: 
![image](https://github.com/lerfast/m4capstone-catalog-of-my-things-/assets/117373513/0d77a33f-5784-4800-975a-ba77b9dd3533)